### PR TITLE
F/form description

### PR DESCRIPTION
**Pull request by amrutha-rajiv**
##### **Description**

- Added description to form control
- Modified DOM of NovoControlElement to support styling

##### **Reviewers**
* @jgodi
* @more

### CODE REVIEW

File: src/elements/form/Control.ts [hunk 1]
```diff
@@ -28,90 +28,103 @@ import { Helpers } from './../../utils/Helpers';
...
-                    <novo-tiles *ngSwitchCase="'tiles'" [options]="control.options" [formControlName]="control.key" (onChange)="modelChange($event)"></novo-tiles>
-                    <!--Picker-->
-                    <div class="novo-control-input-container" *ngSwitchCase="'picker'">
-                        <novo-picker [config]="control.config" [formControlName]="control.key" [placeholder]="control.placeholder" [appendToBody]="control.appendToBody" [parentScrollSelector]="control.parentScrollSelector" *ngIf="!control.multiple" (select)="modelChange($event);" (typing)="handleTyping($event)" (focus)="handleFocus($event)" (blur)="handleBlur($event)"></novo-picker>
-                        <chips [source]="control.config" [type]="control.config.type" [formControlName]="control.key" [placeholder]="control.placeholder" *ngIf="control.multiple" [closeOnSelect]="control.closeOnSelect" (changed)="modelChange($event)" (typing)="handleTyping($event)" (focus)="handleFocus($event)" (blur)="handleBlur($event)"></chips>
+                    <!--Error Message-->
+                    <div class="field-message" *ngIf="errors || showCount || control.description || !condensed">
```
Comment (jgodi) [inline src/elements/form/Control.ts]: Condensed should win here, everything on the Forms look good, but the controls in the Tables use condense to hide all messages under the control. They appear now.

screenshot 2017-02-07 12 03 25

```
<td *ngFor="let column of columns" [attr.data-automation-id]="column.id || column.name" [class.novo-form-row]="editable">
  <novo-table-cell *ngIf="!row._editing[column.name]" [hasEditor]="editable" [column]="column" [row]="row" [form]="tableForm.controls.rows.controls[i]"></novo-table-cell>
  <novo-control *ngIf="row._editing[column.name]" condensed="true" [form]="tableForm.controls.rows.controls[i]" [control]="row.controls[column.name]"></novo-control>
</td>
```

Comment (jgodi) [inline src/elements/form/Control.ts]: To view it in the Tables, go to the Table demo and its the second to last one.

Comment (amrutha-rajiv) [inline src/elements/form/Control.ts]: @jgodi i meant to set up field-message so that it appears at all times except for if not condensed. will fix that

Comment (amrutha-rajiv) [inline src/elements/form/Control.ts]: @jgodi it's fixed now

**Pull request merged**